libclc/generic/libspirv/atomic/atomic_load.cl

```diff
@@ -18,7 +18,7 @@ TYPE __clc__atomic_##PREFIX##load_##AS##_##BYTE_SIZE##_##MEM_ORDER(volatile AS c
   FDECL(TYPE, PREFIX, AS, BYTE_SIZE, acquire)                                                                     \
   FDECL(TYPE, PREFIX, AS, BYTE_SIZE, seq_cst)                                                                     \
   _CLC_DEF TYPE                                                                                                   \
-      _Z18__spirv_AtomicLoadPU3##AS_MANGLED##K##TYPE_MANGLED##N5__spv5Scope4FlagENS1_19MemorySemanticsMask4FlagE( \
+      _Z18__spirv_AtomicLoadP##AS_MANGLED##K##TYPE_MANGLED##N5__spv5Scope4FlagENS1_19MemorySemanticsMask4FlagE( \
```

Comment:
Formatting looks like it could be improved - that `\` should be aligned with the others

Reply:
I've aligned the `\` at the end of the lines, thank you